purchases/src/test/java/com/revenuecat/purchases/models/PeriodTest.kt

```diff
@@ -9,7 +9,7 @@ import org.junit.runner.RunWith
 @RunWith(AndroidJUnit4::class)
 class PeriodTest {
     companion object {
-        private val MAX_OFFSET = Offset.offset(0.0000001)
+        private val MAX_OFFSET = Offset.offset(0.0001)
```

Comment:
don't love reducing accuracy, but having a value this precise would probably risk our tests failing from floating point precision errors